### DISCUSSION TITLE
Shift + Delete = Permanently Delete a File

### DISCUSSION THREAD

**Discussion by RubicBG** [moudey/Shell#223] [Ideas]
It would be great if the Shell program could have a feature that enables users to permanently delete a file when they hover the mouse over the Delete option and press the SHIFT key. This feature can provide an extra layer of safety while deleting files.

Please consider adding this feature in the next release of the Shell program.

**Comment by hiranokite** [comment]
wait, but isn't that standard in windows?

**Comment by RubicBG** [comment]
I saw that the commands from the terminal menu have a tooltip and I thought it would be a good idea to have a tooltip on the delete menu as well. It didn't occur to me at all to test whether the "delete permanently" functionality exists at all. My Bad!  :)

```
shell
{
	static
	{
		code: item(find='Delete' tip=["\xE1A7 Press SHIFT key to permanently delete selected file(s)", tip.warning, 1.0])
	}
}
```

**Comment by Syuanz** [comment]
https://zj.syuanz.top/1822.html

**Comment by Syuanz** [comment]
This system has this feature
